dbus-service.py

```diff
@@ -30,8 +30,8 @@
 
 from pyndn import Name, Face
 
-from eos_data_distribution.ndn import Consumer
-from eos_data_distribution.names import SUBSCRIPTIONS_INSTALLED
+from eosdatadistribution.ndn import Consumer
+from eosdatadistribution.names import SUBSCRIPTIONS_INSTALLED
```

Comment:
As discussed on IRC, as far as I know that’s not possible: Python prefers names without underscores; Debian prohibits them in package names; `pybuild` makes it painful to do anything except have the Python and Debian names equal.